app/__init__.py

```diff
@@ -12,3 +13,11 @@
 
 # Import all models here for easier importing.
 from app.users.models import User
+from app.users.resources import UserListResource, UserResource
+
+api = Api(app)
+
+
+api.add_resource(UserListResource, '/users')
```

Comment:
We should also the url `/users/`. Right now if someone goes to /users/ it would throw a not found error.